src/main.rs

```diff
@@ -81,14 +81,15 @@ fn watch() -> notify::Result<()> {
     let mut watcher: RecommendedWatcher = Watcher::new(tx, Duration::from_secs(2))?;
     watcher.watch("./exercises", RecursiveMode::Recursive)?;
 
-    let _ignored = verify();
+    let _ignored = verify(None);
 
     loop {
-        match rx.recv() {
+        match rx.recv() {     
```

Comment:
could you trim the whitespace here? maybe running `cargo fmt` does this automatically, too